instructor/function_calls.py

```diff
@@ -264,7 +264,7 @@ def from_response(
             Mode.warn_mode_functions_deprecation()
             return cls.parse_functions(completion, validation_context, strict)
 
-        if mode in {Mode.TOOLS, Mode.MISTRAL_TOOLS}:
+        if mode in {Mode.TOOLS, Mode.MISTRAL_TOOLS, Mode.STRUCTURED_OUTPUTS}:
```

Comment:
why does structured outputs using parse tools?

Reply:
They've enabled structured output support for function calling if we use their strict:true https://platform.openai.com/docs/guides/function-calling. That's why we use the pydantic helper in order to enerate the relevant schema.

In this case, I tried using the `.parse` method and found it difficult to figure out how to do validation with + support the partials and iterables that we had already supported. The Function Calling approach was a lot more simple and easy to understand